tests/app/questionnaire/routing/test_operators.py

```diff
@@ -330,3 +330,12 @@ def test_nonetype_operands_for_comparison_operators(operator_name, operands):
 def test_nonetype_operands_for_array_operators(operator_name, operands):
     operator = Operator(operator_name)
     assert operator.evaluate(operands) is False
+
+
+@pytest.mark.parametrize(
+    "operands, expected_result",
+    [([("tuple element 1", "tuple element 2")], 2)],
```

Comment:
As discussed, this should test a list as well. List covers answers, the tuple covers raw schema lists.